ImageNet/train/dawn_4_machine_30min.json

```diff
@@ -0,0 +1,43 @@
+{
+    "version": "v1.0",
+    "author": "Andrew Shaw, Yaroslav Bulatov, Jeremy Howard",
+    "authorEmail": "andrew.t.shaw@gmail.com",
+    "framework": "ncluster / pytorch",
+    "codeURL": "https://github.com/diux-dev/imagenet18/blob/master/training/train_imagenet_nv.py",
```

Comment:
Please include a commit hash. You can either update the `codeURL` field or add a separate `commitHash` field.